packages/block-editor/src/components/rich-text/editable.js

```diff
@@ -119,6 +119,14 @@ export default class Editable extends Component {
 			this.editorNode.className = classnames( nextProps.className, CLASS_NAME );
 		}
 
+		if ( ! isEqual( this.props.start, nextProps.start ) ) {
```

Comment:
These are just strings, right? So we can just check for strict equality?

Reply:
So we can just use `this.props.start === nextProps.start`?

Reply:
Yes :)

Reply:
Done.